packages/babel-preset/src/transformPlugin.test.ts

```diff
@@ -15,6 +15,11 @@ pluginTester({
       plugins: ['typescript'],
     },
   },
+  pluginOptions: {
+    babelOptions: {
+      presets: ['@babel/typescript'],
+    },
+  },
```

Comment:
The preset is added as a config option of the plugin.

Reply:
I guess normally apps written in TS would have the preset for TS anyway in their babel setup?

Reply:
Yes, that's correct. That's why it was there, but as this dependency is implicit it's not the best idea to have it in this way.

This will be documented later properly.